src/api/todos.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (uri: string) => {
+  return client.get<Todo[]>(`/todos${uri}`);
+};
```

Comment:
```suggestion
export const getTodos = (userId: string) => {
  return client.get<Todo[]>(`/todos?userId=${userId}`);
};
```
In this way :arrow_up: